h2/src/main/org/h2/expression/function/table/CSVReadFunction.java

```diff
@@ -43,8 +44,10 @@ public ResultInterface getValue(SessionLocal session) {
             String fieldDelimiter = getValue(session, 4);
             String escapeCharacter = getValue(session, 5);
             String nullString = getValue(session, 6);
+            String quotedNulls = getValue(session, 7);
```

Comment:
This block is for legacy syntax, you shouldn't change it.

New parameters should be passed as options.

Reply:
Removed.